2022-12-21 14:31:00,026 [MainThread  ] [INFO ]  Collecting metrics
2022-12-21 14:31:00,026 [MainThread  ] [DEBUG]  Processing Job example
2022-12-21 14:31:00,027 [MainThread  ] [INFO ]  Getting jobs for example
2022-12-21 14:31:00,027 [MainThread  ] [DEBUG]  Making request to url http://192.168.3.100:8080/job/example/api/json?tree=jobs[name]
2022-12-21 14:31:00,029 [MainThread  ] [DEBUG]  Starting new HTTP connection (1): 192.168.3.100:8080
2022-12-21 14:31:00,284 [MainThread  ] [DEBUG]  http://192.168.3.100:8080 "GET /job/example/api/json?tree=jobs%5Bname%5D&tree=jobs%5Bname%5D HTTP/1.1" 404 464
2022-12-21 14:31:00,286 [MainThread  ] [DEBUG]  Completed request in 0.2590086269992753
2022-12-21 14:31:00,286 [MainThread  ] [ERROR]  Failed collecting metrics Call to url http://192.168.3.100:8080/job/example/api/json?tree=jobs[name] failed with status: 404
Traceback (most recent call last):
  File "/home/caolv/github.com_return200-ok/jenkins-crawler/jenkins_collector.py", line 36, in update_metrics
    branches = self._jenkins.get_jobs(repository.name)
  File "/home/caolv/github.com_return200-ok/jenkins-crawler/jenkins_client.py", line 21, in get_jobs
    response = self._make_request(url_job, params)
  File "/home/caolv/github.com_return200-ok/jenkins-crawler/jenkins_client.py", line 71, in _make_request
    raise Exception(
Exception: Call to url http://192.168.3.100:8080/job/example/api/json?tree=jobs[name] failed with status: 404
2022-12-21 14:31:00,287 [MainThread  ] [INFO ]  Completed collecting metrics in 0.2610011100769043 s
